mods/creative/init.lua

```diff
@@ -80,41 +91,34 @@ local trash = minetest.create_detached_inventory("creative_trash", {
 })
 trash:set_size("main", 1)
 
-creative_inventory.set_creative_formspec = function(player, start_i, pagenum)
+creative_inventory.set_creative_formspec = function(player, start_i, pagenum, tab_idx)
```

Comment:
`tab_idx` should default to 1 ("All") if not specified. Add this after this line:

```lua
tab_idx = tab_idx or 1
```